mcs/build/library.make

```diff
@@ -300,19 +301,19 @@ endif
 # TODO: depend on all *.sources for now and figure out how to list only needed files later
 PROFILE_sources = $(wildcard *.sources)
 
-gensources = $(topdir)/build/gensources.exe
-$(gensources): $(topdir)/build/gensources.cs
-	$(BOOTSTRAP_MCS) -noconfig -debug:portable -r:mscorlib.dll -r:System.dll -r:System.Core.dll -out:$(gensources) $(topdir)/build/gensources.cs
+GENSOURCES_LIBDIR = $(topdir)/class/lib/$(BUILD_TOOLS_PROFILE)
+GENSOURCES_CS = $(topdir)/build/gensources.cs
+GENSOURCES_EXE = $(topdir)/build/gensources.exe
```

Comment:
This should go to `$(topdir)/build/...` as everything else

Reply:
it's not a destination folder it's where it's referencing the libraries from for build purposes. i can't move it